CODE_OF_CONDUCT.md

```diff
@@ -23,13 +23,13 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
- advances
+  advances
```

Comment:
@vzajkov looks like this file got added. You might need to just revert this change.